src/convertToObject.js

```diff
@@ -1,18 +1,29 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString
+    .split(';\n')
+    .map(str => str.split(':'))
+    .filter(arr => arr.length === 2)
+    .map((arr) => {
+      const [ a, b ] = arr;
+
+      return [a.trim(), b.trim()];
+    })
+    .reduce((prev, item) => {
+      return {
+        ...prev,
+        [item[0]]: item[1],
+      };
+    }, {});
```

Comment:
you can make it more shorter)
```suggestion
    .map(([ a, b ]) => [a.trim(), b.trim()])
    .reduce((prev, [key, value]) => ({
        ...prev,
        [key]: value,
      }), {});
```